exportTestEnvironments.js

```diff
@@ -0,0 +1,43 @@
+import dotenv from 'dotenv';
+import * as childProcess from "child_process";
+import minimist from 'minimist';
+
+const { i, s, t } = minimist(process.argv.slice(2));
+
+dotenv.config();
+
+const { API_KEY, SYNC_SOURCE_TEST_ENVIRONMENT_ID, SYNC_TARGET_TEST_ENVIRONMENT_ID, EXPORT_IMPORT_TEST_DATA_ENVIRONMENT_ID } = process.env;
```

Comment:
nitpick, since it's just an export script. but maybe error handling for missing env variables?